static-website/lambdaedge-index-document.yaml

```diff
@@ -66,7 +66,7 @@ Resources:
         # If you change the ZipFile, rename the logical id LambdaVersionV4 to trigger a new version creation!
```

Comment:
could you rename the logical id as described here. Otherwise existing stacks will not catch up the change because of Lambda Versions

Reply:
increased V.. number